src/hotspot/share/cds/filemap.cpp

```diff
@@ -2465,6 +2504,14 @@ bool FileMapHeader::validate() {
     return false;
   }
 
+  if (compact_headers() != UseCompactObjectHeaders) {
+    log_info(cds)("The shared archive file's UseCompactObjectHeaders setting (%s)"
```

Comment:
Printing on the `info` level mimics what we do when there's a mismatch for compressed classes (and oops), but I wonder if that one is intentional or if it is accidentally printing to 'info' instead of 'warning'.

Reply:
@iklam informed me that some of the info levels (including this line) should be converted to warning.

Reply:
Yeah that looks inconsistent with other places where we print a warning instead. I'll change it to warning for the UCOH check.